source/index.html

```diff
@@ -159,6 +163,7 @@ <h2 class="main-footer__title">#visitsedona</h2>
       </a>
     </footer>
   </div>
+  <script src="script.js"></script>
```

Comment:
```suggestion
  <script src="js/script.js"></script>
```
Иначе путь к файлу недействительный и код JS не работает.

На страницах каталога и формы также нужно подключить скрипт.